### DISCUSSION TITLE
How to horizontally slice a single .png  file into 10 .png files?

### DISCUSSION THREAD

**Discussion by gitmhe43** [ImageMagick/ImageMagick#7514] [Help] [answered]
### ImageMagick version

7.0.11 Q16-HDRI (64 bit)

### Operating system, version and so on

Windows 10 Pro

### Description

How to slice a 720x10000 .png file (File X) into ten 720x1000 .png files, slicing horizontally from the top, such that File1.png holds the top 720x1000 slice, File2.png holds the second 1000 pixel slice (720,1000 to 720,1999), and so on?

Thanks you!

**Comment by snibgo** [comment, marked as answer]
See yesterday's thread https://github.com/ImageMagick/ImageMagick/discussions/7512

**Reply by gitmhe43** [reply]
Thank you, snibgo.

That worked!
